Story/0AllStories.cs

```diff
@@ -709,7 +706,12 @@ public void CompleteAll()
         UnderGroundLab.partofundergroundlabb();
         Core.Logger($"Story: Underground Lab - Complete?");
 
-
+        if (Core.IsMember)
+        {
+            Voltaire.DoAll();
+            Core.Logger($"Story: Voltaire Quests - Complete");
```

Comment:
Saga: Voltaire Quests - Complete
and it should be placed among saga in core files not in standalone region

Reply:
Ok